### ISSUE TITLE
Error valid-path-segments on path /gps/{latitude},{longitude}

### ISSUE THREAD

**Issue by jfoclpf** [IBM/openapi-validator#505]
I read the [docs](https://github.com/IBM/openapi-validator/blob/main/docs/ibm-cloud-rules.md#rule-valid-path-segments) (Rule: valid-path-segments) but I don't understand this error. That's the only error I'm having in my openapi file.

Imagine I have a path to get a point in a map given some coordinates. Why the path ` /gps/{latitude},{longitude}` should be wrong if both `latitude` and `longitude` have the type `number`?

For example: https://geoapi.pt/gps/40.153687,-8.514602

I mean, is the comma `,` not a valid path segment separator?

(after thinking a bit)

Wait, do you want me to do something like ` /gps/{latitude_longitude}`? But then I cannot have `latitude` with type `number`

**Comment by jfoclpf** [comment]
this is the rule in the file

```yaml
  /gps/{latitude},{longitude}:
    summary: Distrito, Município e Freguesia através de coordenadas GPS
    get:
      parameters:
        - in: path
          name: latitude
          required: true
          schema:
            type: number
            example: 40.153687
          description: Latitude da coordenada
        - in: path
          name: longitude
          required: true
          schema:
            type: number
            example: -8.514602
          description: Longitude da coordenada
        - $ref: '#/components/parameters/json'
      responses:
        '200':
          description: Distrito, Município e Freguesia através de coordenadas GPS
          content:
            application/json:
              schema:
                type: string
```

**Comment by dpopp07** [comment]
It seems like path parameters are actually not well suited to your use case. Path parameters really should be unique identifiers for a resource within your API.

Is there a reason why you aren't using query parameters for latitude and longitude here?

**Comment by jfoclpf** [comment]
I also allow the possibility of query parameters, but I find `/gps/40.153687,-8.514602` nicer to present than `/gps?lat=40.153687&lon=-8.514602`.

Anyway I overcame the problem by having a `/gps/{coordinates}` with type string (for example `'40.153687,-8.514602'`)

**Comment by jfoclpf** [comment]
Thanks a lot
